request-manager/resources/request_manager_schema.sql

```diff
@@ -41,3 +42,14 @@ CREATE TABLE IF NOT EXISTS `job_log` (
 
   PRIMARY KEY (`request_id`, `job_id`, `try`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+CREATE TABLE IF NOT EXISTS `suspended_job_chains` (
+  `request_id`          BINARY(20) NOT NULL,
+  `suspended_job_chain` LONGBLOB NOT NULL,
+  `rm_host`             VARCHAR(64) NULL DEFAULT NULL,
+  `last_updated_at`     TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```

Comment:
Typically, this is called just `update_at`. "Last" is implied.

Reply:
`update_at` or `updated_at`?